jade/page-contents/chips_content.html

```diff
@@ -76,35 +76,46 @@ <h3 class="header">Initialization</h3>
         <pre><code class="language-javascript">
   document.addEventListener('DOMContentLoaded', function() {
     var elems = document.querySelectorAll('.chips');
-    var instances = M.Chips.init(elems, options);
+    var instances = M.Chips.init(elems, {
+      // specify options here
+    });
   });
 
   // Or with jQuery
-
-  $('.chips').chips();
-  $('.chips-initial').chips({
-    data: [{
-      tag: 'Apple',
-    }, {
-      tag: 'Microsoft',
-    }, {
-      tag: 'Google',
-    }],
+  $(document).ready(function(){
+    $('.chips').chips({
+      // specify options here
+    })
+  });
+  $(document).ready(function(){
+    $('.chips-initial').chips({
+      data: [{
+        tag: 'Apple',
+      }, {
+        tag: 'Microsoft',
+      }, {
+        tag: 'Google',
+      }],
+    })
   });
-  $('.chips-placeholder').chips({
-    placeholder: 'Enter a tag',
-    secondaryPlaceholder: '+Tag',
+  $(document).ready(function(){
+    $('.chips-placeholder').chips({
+      placeholder: 'Enter a tag',
+      secondaryPlaceholder: '+Tag',
+    })
   });
-  $('.chips-autocomplete').chips({
-    autocompleteOptions: {
-      data: {
-        'Apple': null,
-        'Microsoft': null,
-        'Google': null
-      },
-      limit: Infinity,
-      minLength: 1
-    }
+  $(document).ready(function(){
+    $('.chips-autocomplete').chips({
+      autocompleteOptions: {
+        data: {
+          'Apple': null,
+          'Microsoft': null,
+          'Google': null
+        },
+        limit: Infinity,
+        minLength: 1
+      }
+    })
   });
```

Comment:
the chips.... oh no..... this is not consistent with the non jquery one 😞 

---

btw IMO, the jquery one should merged as one `$(document).ready` instead. So the new user will not stupidly read this as "oh, I need multiple document ready for these".

Reply:
maybe also copy the jquery options to the non jquery? How about it? @DanielRuf @doughballs

Reply:
Makes sense yes (to cpy them to the non jQuery version).

I'll give you write permissions so you can help with this PR if you want.

Reply:
> the chips.... oh no..... this is not consistent with the non jquery one 😞 
> 
> 
> 
Yea I saw this too, but I thought it might have been intentional. 

I'm busy at the moment but i might send an update when I get the chance.

Reply:
> I'll give you write permissions so you can help with this PR if you want.

@Smankusors you should have received an invitation for the members with write access group. So you should be able to push changes if you want to.

> I'm busy at the moment but i might send an update when I get the chance.

No hurry. And if you want we can help with this.

Reply:
By all means! 

I honesty just wanted to add the one 
var options = {} 
line to the docs haha

But I'm glad I can help make the documentation better!

Reply:
@DanielRuf hmm thanks for the trust, but unfortunately I'm busy right now to write another code. But I still can do code review for now, and merge this PR when it's done 😅

Reply:
Sure, no hurry =)